src/functions/FilterResByPrice.js

```diff
@@ -1,18 +1,19 @@
 
 const FilterResByPrice = (results) => {
   const filteredResults = [[], [], []]
-  const length = results.length
   let i = 0
 
-  for (i = 0; i < length; i++) {
-    (results[i].price === '$' || results[i].price === '€')
-      ? filteredResults[0].push(results[i]) 
-      : (results[i].price === '$$' || results[i].price === '€€')
-        ? filteredResults[1].push(results[i]) 
-        : filteredResults[2].push(results[i])
+  for (i = 0; i < results.length; i++) {
+    if (results[i].price === '$' || results[i].price === '€') {
+      filteredResults[0].push(results[i])
+    } else if (results[i].price === '$$' || results[i].price === '€€') {
+      filteredResults[1].push(results[i])
+    } else {
+      filteredResults[2].push(results[i])
+    }
```

Comment:
FilterResByPrice - change the nested '?' operator to 'if' statement, for more simple and readable code.